solutions/usaco-965.mdx

```diff
@@ -88,4 +91,48 @@ int main() {
 		cout << cows[t] << '\n';
 	}
 }
-```
\ No newline at end of file
+```
+</CPPSection>
+
+<PySection>
+## Brute Force with Itertools
+
+<Warning>
+Due to the brute force nature of this solution, it is very slow and is not the optimal solution.
+</Warning>
```

Comment:
Do we really need this?

Reply:
yeah i think this can be removed

Reply:
wait nooo

Reply:
i think we should keep

Reply:
We should remove b/c the previous solution states "What if there were up to $10^5$ cows? Then neither of the solutions provided in the analysis would be fast enough." Imo, this is pretty redundant, so please remove.

Reply:
I think if you change it to an info section and then say something along the lines of "This solution can be done in {actual tc}"